earth_enterprise/src/fusion/gecombineterrain/combineterrain.cpp

```diff
@@ -187,6 +191,7 @@ void TerrainCombiner::WriteCombinedTerrain(
     const QuadtreePath even_path,
     const std::vector<const TerrainPacketItem *> &items,
     uint progress_increment) {
+  BEGIN_PROFILING(calcReadSize, "calcReadSize", even_path.AsString());
```

Comment:
may be use more descriptive name for the operation instead of short name "calcReadSize" ?